.github/composites/docker-build-push/action.yaml

```diff
@@ -0,0 +1,129 @@
+name: 'Build and Publish Docker Image'
+description: 'Build and Publish Docker Image'
+inputs:
+  custodian_image_var:
+    description: 'env var name for custodian image'
+    required: true
+  dockerfile:
+    description: 'Dockerfile to build'
+    required: true
+  repository:
+    description: 'docker hub repository'
+    required: true
+  image:
+    description: 'image'
+    required: true
+  name:
+    description: 'image name'
```

Comment:
is this value different from docker file?

Reply:
i've simplified all of the vars now